lib/contestant.rb

```diff
@@ -0,0 +1,32 @@
+class Contestant
+  attr_reader :first_name,
+              :last_name,
+              :full_name,
+              :age,
+              :state_of_residence,
+              :spending_money,
+              :game_interests
+
+  def initialize(contestant_info)
+    @contestant_info = contestant_info
```

Comment:
We don't need `@contestant_info` stored as an instance variable; we just need to use `contestant_info` to create the other instance variables